storage/scylla_config.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef TA_SCYLLA_CONFIG_H_
+#define TA_SCYLLA_CONFIG_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define SCYLLA_DEFAULT_HOST "node0.puyuma.org"
+#define IRI_DEFAULT_HOST "node1.puyuma.org"
```

Comment:
I dislike the hardcoded hostname. `node0` might retire soon.

Reply:
We may need to remove this and use CLI arguments to assign values to them.